src/daemon/bitcoind/poller.rs

```diff
@@ -762,6 +887,21 @@ fn comprehensive_rescan(
                     &unvault_tx,
                 )?;
 
+                if db_transaction_has_priority_dbtx(db_tx, &unvault_txid)? {
+                    let broadcasted_blockheight = bitcoind.get_tip()?.height;
+                    needs_cpfp_cache.insert(
+                        unvault_txid,
+                        CpfpTxInfo {
+                            broadcasted_blockheight,
+                            feebumped_blockheight: None,
+                        },
+                    );
+                    log::debug!(
+                        "Inserted Unvault transaction {} in CPFPable table as it got unconfirmed (unvault case 3)",
+                        unvault_txid
+                    );
+                }
```

Comment:
Oh, that's very noble but i wouldn't even bother with trying to handle CPFP with reorgs. It's something we can add later and that is going to be a large can of worms if we add now.